requirements.txt

```diff
@@ -3,7 +3,7 @@
 -e git://github.com/rsetaluri/gemstone.git#egg=gemstone
 -e git://github.com/rsetaluri/canal.git#egg=canal
 ordered_set
-magma-lang==0.1.1
-mantle==0.1.2
+magma-lang==0.1.18
+mantle==0.1.14
 cosa==0.2.8
-fault==0.42
+fault==1.0.7
```

Comment:
I propose we relax version requirements on these dependencies by using `0.1.*` or `0.*` (magma/mantle) and `1.0.*` (fault) or `1.*`. Ideally these dependencies should only release changes that don't affect backwards compatibility (following https://semver.org/), so it should make it easier for us to deploy/integrate new features and bug fixes, while avoiding backwards incompatible API changes (through the first version number)

Reply:
See #190